doc/reference/files/config/sandboxing_preference.rst

```diff
@@ -0,0 +1,19 @@
+sandboxing_preference
+---------------------
+
+The preferred sandboxing setting. Individual rules may specify different
```

Comment:
```suggestion
The preferred sandboxing setting. Individual rules may specify different
```
Same as in `jobs` -- perhaps "`sandboxing_preference` is the preferred sandboxing setting"